3-enrich/stream-enrich/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sources/KinesisSource.scala

```diff
@@ -66,7 +82,8 @@ object KinesisSource {
       _ <- KinesisSink.validate(kinesisConfig, config.out.enriched)
       _ <- utils.validatePii(emitPii, config.out.pii)
       _ <- KinesisSink.validate(kinesisConfig, config.out.bad)
-      provider <- getAWSCredentialsProvider(kinesisConfig.aws)
+      sourceProvider <- getAwsCredentialsProvider(kinesisConfig.aws)
+      sinkProvider <- getAWSCredentialsProvider(kinesisConfig.aws)
```

Comment:
not sure I like this naming, seems easy to get confused which of these is which

Reply:
I am going to remove the `getAWSCredentialsProvider` one in the next PR. Snowplow likes one commit per issue and PR.